diff -rupN orig/capi/CMakeLists.txt new/capi/CMakeLists.txt
--- orig/capi/CMakeLists.txt	2014-11-30 15:49:25.078555486 +0100
+++ new/capi/CMakeLists.txt	2014-11-30 16:06:19.134887151 +0100
@@ -33,11 +33,17 @@ if(NOT GEOS_ENABLE_MACOSX_FRAMEWORK)
       VERSION ${CAPI_VERSION}
       CLEAN_DIRECT_OUTPUT 1)
   else()
-    set_target_properties(geos_c
-      PROPERTIES
-      VERSION ${CAPI_VERSION}
-      SOVERSION ${CAPI_SOVERSION}
-      CLEAN_DIRECT_OUTPUT 1)
+    if (ANDROID)
+      set_target_properties(geos_c
+        PROPERTIES
+        CLEAN_DIRECT_OUTPUT 1)
+    else()
+      set_target_properties(geos_c
+        PROPERTIES
+        VERSION ${CAPI_VERSION}
+        SOVERSION ${CAPI_SOVERSION}
+        CLEAN_DIRECT_OUTPUT 1)
+    endif()
   endif()
 
 endif()
diff -rupN orig/src/CMakeLists.txt new/src/CMakeLists.txt
--- orig/src/CMakeLists.txt	2017-02-14 12:27:51.046377391 +0100
+++ new/src/CMakeLists.txt	2017-02-14 12:29:13.280316592 +0100
@@ -67,9 +67,14 @@ else()
     set_target_properties(geos
       PROPERTIES
       DEFINE_SYMBOL GEOS_DLL_EXPORT
-      VERSION ${VERSION}
       CLEAN_DIRECT_OUTPUT 1)
 
+    if(NOT ANDROID)
+      set_target_properties(geos
+        VERSION ${VERSION}
+      )
+    endif()
+
     install(TARGETS geos
       RUNTIME DESTINATION bin
       LIBRARY DESTINATION lib
\ No newline at end of file
