diff -Naur libpng-1.6.37/CMakeLists.txt b/CMakeLists.txt
--- libpng-1.6.37/CMakeLists.txt	2019-04-14 20:10:32.000000000 +0200
+++ b/CMakeLists.txt	2020-06-23 15:46:12.367354764 +0200
@@ -272,7 +272,7 @@
 
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
-if(NOT AWK OR ANDROID)
+if(NOT AWK)
   # No awk available to generate sources; use pre-built pnglibconf.h
   configure_file(${CMAKE_CURRENT_SOURCE_DIR}/scripts/pnglibconf.h.prebuilt
                  ${CMAKE_CURRENT_BINARY_DIR}/pnglibconf.h)
@@ -451,7 +451,7 @@
     "${CMAKE_CURRENT_BINARY_DIR}/scripts/symbols.chk"
     "${CMAKE_CURRENT_BINARY_DIR}/scripts/symbols.out"
     "${CMAKE_CURRENT_BINARY_DIR}/scripts/vers.out")
-endif(NOT AWK OR ANDROID)
+endif(NOT AWK)
 
 # OUR SOURCES
 set(libpng_public_hdrs
