From d5c9efc840605ecdc556e5f72a74172f80d5bd96 Mon Sep 17 00:00:00 2001
From: Denis Rouzaud <denis.rouzaud@gmail.com>
Date: Fri, 14 May 2021 08:01:12 +0200
Subject: [PATCH] python dev not always required

---
 CMakeLists.txt | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f19dfdea9117..8ac5007ac1de 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -939,7 +939,12 @@ set(QGIS_INSTALL_SYS_LIBS TRUE CACHE BOOL "If set to TRUE install all required s
 set(MIN_PYTHON_VERSION "3.7")
 set(Python_FIND_FRAMEWORK "LAST")
 
-find_package(Python ${MIN_PYTHON_VERSION} REQUIRED COMPONENTS Interpreter Development)
+set(QGIS_PYTHON_COMPONENTS "Interpreter")
+if (WITH_DESKTOP OR WITH_BINDINGS)
+  set(QGIS_PYTHON_COMPONENTS "${QGIS_PYTHON_COMPONENTS} Development")
+endif()
+find_package(Python ${MIN_PYTHON_VERSION} REQUIRED COMPONENTS ${QGIS_PYTHON_COMPONENTS})
+
 message("-- Found Python executable: ${Python_EXECUTABLE} (version ${Python_VERSION})")
 message("-- Python library: ${Python_LIBRARIES}")
 message("-- Python site-packages: ${Python_SITEARCH}")
